### ISSUE TITLE
Account linking

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#73]
We should allow different sign in accounts to be linked together.

**Comment by cointastical** [comment]
And here's an example of why this linking is important:

*What if I use Twitter auth for my Stacker.News account, and then Twitter suspends my account?*
https://stacker.news/items/7746

**Comment by huumn** [comment]
Sadly, we'll need to wait on this issue https://github.com/nextauthjs/next-auth/issues/2988 getting implemented before we can proceed because we user their lib for auth with jwt's. 

It looks like maintainers are interested in getting it done but idk when it'll happen.

**Comment by cbergqvist** [comment]
Signed in with Breez on one phone which I'm now trying to migrate off of. Unable to migrate the Breez node to my new phone, so being able to link another lightning login to the same account would be great!

**Comment by huumn** [comment]
I'll check in with where the auth library is at wrt this and see what I can do. Might have to fork it. Lots of people have asked for this.

**Comment by cbergqvist** [comment]
Is linking multiple lightning logins to an account now live on the site? If so, where does one do it, seems I can only link/unlink one lightning auth method.

**Comment by huumn** [comment]
You can't do multiple nodes yet. I'll open a separate issue for it and try to get around to it.
